docs/_user_guide/slurm.md

```diff
@@ -190,49 +185,48 @@ JOBID PARTITION     NAME     USER ST          START_TIME  NODES SCHEDNODES
-You can also cancel all of your running and pending jobs with `scancel -u USERNAME` where `USERNAME` is your username.
+The `scancel JOBID` command will cancel your job.  You can find the unique numeric `JOBID` of your job with `squeue`. You can also cancel all of your running and pending jobs with `scancel -u USERNAME` where `USERNAME` is your username.
 
 ### Constraining my job to specific nodes using node features
 
 Certain nodes may have particular features that your job requires, such
-as a GPU.  These features can be viewed as follows:
+as a GPU. These features can be viewed as follows:
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ sinfo -o "%10N  %5c  %5m  %64f  %10G"
-NODELIST    CPUS   MEMOR  AVAIL_FEATURES                                                    GRES
-yen[11-17]  32+    10315  (null)                                                            (null)
-yen-gpu1    64     25736  GPU_BRAND:NVIDIA,GPU_UARCH:AMPERE,GPU_MODEL:A30,GPU_MEMORY:24GiB  gpu:4
-yen-gpu2    64     25736  GPU_BRAND:NVIDIA,GPU_UARCH:AMPERE,GPU_MODEL:A40,GPU_MEMORY:48GiB  gpu:4
+NODELIST    CPUS   MEMOR  AVAIL_FEATURES                                                    GRES      
+yen[11-18]  32+    10315  (null)                                                            (null)    
+yen-gpu1    64     25736  GPU_BRAND:NVIDIA,GPU_UARCH:AMPERE,GPU_MODEL:A30,GPU_MEMORY:24GiB  gpu:4     
+yen-gpu[2-  64     25736  GPU_BRAND:NVIDIA,GPU_UARCH:AMPERE,GPU_MODEL:A40,GPU_MEMORY:48GiB  gpu:4
 ```
 
-For example, to ensure that your job will run on a node that has an
-NVIDIA Ampere A40 GPU, you can include the `-C`/`--constraint` option to
-the `sbatch` command or in an `sbatch` script.  Here is a trivial
-example command that demonstrates this: `sbatch -C "GPU_MODEL:A30" -G 1 -p gpu --wrap "nvidia-smi"`
+For example, to ensure that your job will run on a node that has an NVIDIA Ampere A40 GPU, you can include the `-C`/`--constraint` option to the `sbatch` command or in an `sbatch` script. Here is a trivial example command that demonstrates this: 
+
+```bash title="sbatch Command"
+sbatch -C "GPU_MODEL:A30" -G 1 -p gpu --wrap "nvidia-smi"
+```
 
 At present, only GPU-specific features exist, but additional node features may be added over time.
 
+## Submit Your First Job to Run on Yen Slurm
 
-## Submit Your First Job to Run on Yen-Slurm
+Let's walk through the submission of a Python script to Yen Slurm.
 
-We are going to copy scripts including example python scripts and Slurm submission scripts.
+First, let's copy some example Python scripts and Slurm submission scripts.
 Make a directory inside your home directory called `intermediate_yens_2023`.
-Then copy the scripts for this class from `scratch` to your `intermediate_yens_2023` working directory so you can modify and run them.
+Then copy the scripts from `scratch` to your `intermediate_yens_2023` working directory so you can modify and run them.
```

Comment:
I think we'll need to change this...maybe just go with a simple python script and delete all the copy of the intermediate yens scripts from scratch etc